bspump/analyzer/timewindowmatrix.py

```diff
@@ -190,8 +190,8 @@ def advance(self, target_ts):
 		return added
 
 
-	def close_row(self, row_index, clear=True):
-		super().close_row(row_index, clear=clear)
+	def close_row(self, row_index):
```

Comment:
I propose to revert it and instead fix the parent class `NamedMatrix`:
```
	def close_row(self, row_index, clear=True):
		super().close_row(row_index, clear=clear)

```

Reply:
Agree with @margirova